nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/registry/flow/mapping/NiFiRegistryFlowMapper.java

```diff
@@ -185,6 +186,11 @@ public InstantiatedVersionedProcessGroup mapProcessGroup(final ProcessGroup grou
         });
     }
 
+    // This is specific for the {@code NifiRegistryFlowRegistryClient}
+    private String getRegistryUrl(final FlowRegistryClientNode registry) {
+        return registry.getComponentType().equals("org.apache.nifi.registry.flow.NifiRegistryFlowRegistryClient") ? registry.getRawPropertyValue(registry.getPropertyDescriptor("URL")) : "";
```

Comment:
Might be worth leaving a comment in the code here that we do this purely for backward compatibility purposes. Otherwise, we'll be scratching our heads at this later :)